Added mobius inversion

Chillee opened the pull request:
I'm not sure what you meant by this: https://github.com/kth-competitive-programming/kactl/pull/159#discussion_r388126994

I'm not aware of such a version - are you thinking of what he reduced the problem to here? https://codeforces.com/blog/entry/53925

Looks like this: 

![image](https://user-images.githubusercontent.com/6355099/79252471-abea5a00-7e4f-11ea-9c3f-9fae6f52357f.png)

simonlindholm commented:
> I'm not sure what you meant by this: #159 (comment)

https://en.wikipedia.org/wiki/M%C3%B6bius_inversion_formula#Generalizations

simonlindholm commented on content/number-theory/chapter.tex:
```diff
@@ -48,3 +48,14 @@ \section{Estimates}
+
+  Other useful formulas/forms:
+  \[ \sum_{d | n} \mu(d) = [ n = 1] \]
+  \[ g(n) = \sum_{n|d} f(d) \Leftrightarrow f(n) = \sum_{n|d} g(d)\mu(d/n)\]
```
`g(n) = \sum_{n|d} f(d)` seems to be a sum over infinitely many d?

Chillee replied:
https://math.stackexchange.com/questions/1965342/does-the-mobius-inversion-theorem-hold-when-sum-function-is-over-multiples-inste

and

https://codeforces.com/blog/entry/59962?#comment-475400

It's possible if `f(d) = 0` for some high enough d.

simonlindholm replied:
Oh, I see. I wonder if it's worth a comment, but not obvious what would fit. "(if convergent)" maybe

Chillee commented:
Added the floor formula. Also switched to `$` instead of `\[` since it took up a lot of space. I don't know if there's a better way to do it.

Current:

![image](https://user-images.githubusercontent.com/6355099/79949147-14829980-8443-11ea-8bfb-3d85c2844c0b.png)

Originally:

![image](https://user-images.githubusercontent.com/6355099/79949269-4562ce80-8443-11ea-82a7-cdf35a6e66cd.png)

simonlindholm commented on content/number-theory/chapter.tex:
```diff
@@ -48,3 +48,17 @@ \section{Estimates}
+  \[ g(n) = \sum_{d|n} f(d) \Leftrightarrow f(n) = \sum_{d|n} \mu(d)g(n/d) \]
+  Other useful formulas/forms:
+
+  $ \sum_{d | n} \mu(d) = [ n = 1] $ (very useful)
```
remove this comment since it line-wraps, I guess
